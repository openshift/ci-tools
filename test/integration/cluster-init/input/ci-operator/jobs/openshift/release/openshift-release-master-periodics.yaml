periodics:
- agent: kubernetes
  cluster: app.ci
  decorate: true
  extra_refs:
    - base_ref: master
      org: openshift
      repo: release
  interval: 12h
  labels:
    ci.openshift.io/generator: prowgen
    ci.openshift.io/role: infra
  name: periodic-openshift-release-master-build01-apply
  spec:
    containers:
      - args:
          - --config-dir=clusters/build-clusters/build01
          - --as=
          - --KUBECONFIG=/etc/build-farm-credentials/KUBECONFIG
          - --confirm=true
        command:
          - applyconfig
        image: applyconfig:latest
        imagePullPolicy: Always
        name: ""
        resources:
          requests:
            cpu: 10m
        volumeMounts:
          - mountPath: /etc/build-farm-credentials
            name: build-farm-credentials
            readOnly: true
    serviceAccountName: config-updater
    volumes:
      - name: build-farm-credentials
        secret:
          items:
            - key: sa.config-updater.build01.config
              path: kubeconfig
          secretName: config-updater