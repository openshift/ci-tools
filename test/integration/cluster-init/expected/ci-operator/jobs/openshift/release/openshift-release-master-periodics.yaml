periodics:
- agent: kubernetes
  cluster: app.ci
  decorate: true
  extra_refs:
  - base_ref: master
    org: openshift
    repo: release
  interval: 12h
  labels:
    ci.openshift.io/role: infra
  name: periodic-openshift-release-master-build01-apply
  spec:
    containers:
    - args:
      - --config-dir=clusters/build-clusters/build01
      - --as=
      - --KUBECONFIG=/etc/build-farm-credentials/KUBECONFIG
      - --confirm=true
      command:
      - applyconfig
      image: applyconfig:latest
      imagePullPolicy: Always
      name: ""
      resources:
        requests:
          cpu: 10m
      volumeMounts:
      - mountPath: /etc/build-farm-credentials
        name: build-farm-credentials
        readOnly: true
    serviceAccountName: config-updater
    volumes:
    - name: build-farm-credentials
      secret:
        items:
        - key: sa.config-updater.build01.config
          path: kubeconfig
        secretName: config-updater
- agent: kubernetes
  cluster: app.ci
  decorate: true
  extra_refs:
  - base_ref: master
    org: openshift
    repo: release
  interval: 12h
  labels:
    ci.openshift.io/role: infra
  name: periodic-openshift-release-master-newCluster-apply
  spec:
    containers:
    - args:
      - --config-dir=clusters/build-clusters/newCluster
      - --as=
      - --KUBECONFIG=/etc/build-farm-credentials/KUBECONFIG
      - --confirm=true
      command:
      - applyconfig
      image: applyconfig:latest
      imagePullPolicy: Always
      name: ""
      resources:
        requests:
          cpu: 10m
      volumeMounts:
      - mountPath: /etc/build-farm-credentials
        name: build-farm-credentials
        readOnly: true
    serviceAccountName: config-updater
    volumes:
    - name: build-farm-credentials
      secret:
        items:
        - key: sa.config-updater.newCluster.config
          path: kubeconfig
        secretName: build-farm-credentials
